src/main/java/edu/cornell/kfs/pdp/service/impl/CuAchServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package edu.cornell.kfs.pdp.service.impl;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.kuali.kfs.krad.service.BusinessObjectService;
+import org.kuali.kfs.pdp.PdpPropertyConstants;
+import org.kuali.kfs.pdp.businessobject.PayeeACHAccount;
+import org.kuali.kfs.pdp.service.impl.AchServiceImpl;
+
+import edu.cornell.kfs.pdp.service.CuAchService;
+
+public class CuAchServiceImpl extends AchServiceImpl implements CuAchService {
+    private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(CuAchServiceImpl.class);
+
+    private BusinessObjectService businessObjectService;
+
+    /*
+     * This implementation is nearly identical to that from the AchServiceImpl.getAchInformation method.
+     * The only differences are that the fields map does not add an entry for the active indicator,
+     * and that the logging statements have been updated accordingly to match the new method name.
+     */
```

Comment:
I think this comment could be removed and the method name updated to something along the lines of getActiveAndInactiveAchInformation() or getAllAchInformation()

Reply:
Ok, I will see if I can come up with a better method name so that the comment won't be needed.